README.md

```diff
@@ -19,6 +19,25 @@ Because ML practitioners use so many different types of tools, it is a key goal
 
 Ultimately, we want to have a set of simple manifests that give you an easy to use ML stack _anywhere_ Kubernetes is already running and can self configure based on the cluster it deploys into.
 
+
+## Who should consider using Kubeflow?
+
+Based on the current functionality you should consider using Kubeflow if
+
+  * You want to train/serve TensorFlow models in different environments (e.g. local, on prem, and cloud)
+  * You want to use Jupyter notebooks to manage TensorFlow training jobs
+       * kubeflow is particularly helpful if you want to launch training jobs that use more resources (more nodes or more GPUs) then your notebook.
+  * You want to combine TensorFlow with other processes
+       * For example if you want to use [tensorflow/agents](https://github.com/tensorflow/agents) to run simulations to generate data for training
+         reinforcement learning models
+
```

Comment:
"We should note, the above list covers ONLY currently available functionality. We hope to expand 'Who should consider using Kubeflow' very soon! (See the next section)"

Reply:
PTAL

Reply:
It would be good to have a list of functionality that we would like it to have. For example:

+ Run Kaggle kernels at scale

+ Deploy secure web services from Jupyter using Cloud Endpoints

+ etc.

This would serve as a pool of ideas that contributors could work on.

In general, though, this is a big step in the right direction.

Reply:
"Functionality wish list"

Reply:
I think the wish list is GitHub issues.

Reply:
Good point :)